### ISSUE TITLE
Email notifications on admin activity

### ISSUE THREAD

**Issue by chapmanio** [o2Labs/office-booker#53]
When certain admin activity is made on behalf of another user, that user should receive some form of notification.  

This could be a banner the next time they load the app, but in the case of new admin users, etc. the most comprehensive option would be via email:

- New booking (sent to user the booking has been made for)
- Cancelled booking (sent to the user who's booking has been cancelled)
- New user (probably just for System or Office admin users)

Labelled as a question as considerations should be made on volume of emails for both cost and spam considerations.

**Comment by danielrbradley** [comment]
Yup, good question ... the action is currently required to be actioned manually by the personal making the change to send the relevant communications.

**Comment by chapmanio** [comment]
A request has come in for email notifications on booking creation - which could tie in to this as well.

**Comment by danielrbradley** [comment]
Yup, think the booking related email notifications are worth exploring in more detail.

Using the ICS format, we can send a VEVENT with a UID, then if the booking is cancelled, we should be able send another ICS attachment with the same UID with `METHOD:CANCEL` and `STATUS:CANCELLED`.
